tests/Library/Vanilla/EmbeddedContent/Embeds/TwitchEmbedFilterTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Library\EmbeddedContent\Embeds;
+
+use Vanilla\EmbeddedContent\Embeds\TwitchEmbed;
+use Vanilla\EmbeddedContent\Embeds\TwitchEmbedFilter;
+use Vanilla\EmbeddedContent\Factories\TwitchEmbedFactory;
+use VanillaTests\Fixtures\MockHttpClient;
+use VanillaTests\MinimalContainerTestCase;
+
+/**
+ * Test twitch embed filter.
+ */
+class TwitchEmbedFilterTest extends MinimalContainerTestCase {
```

Comment:
Do we need the container here or can we make it work with a regular ole `TestCase`?